### DISCUSSION TITLE
Request - move taskbar to top

### DISCUSSION THREAD

**Discussion by radixs** [valinet/ExplorerPatcher#246] [Ideas]
Hello, first of all thank you very much for this tool, it saved me having to revert back to win10 (like who decided at MS that replicating macbook interface made for slow people is a good idea... ehhhh).

The request - I prefer to have the taskbar at the top - just because navigation this way is faster (apps have menus at the top, so it is just less mouse moving this way). Would it be possibel to include this feature?

**Comment by valinet** [comment]
Hi

This feature was already supported in Windows 10 with the old taskbar. All this software does is revert the UI back to the one from Windows 10. Thus, this feature is still there.

How did you move the taskbar to top in Windows 10? The same works here: right click the taskbar, uncheck “Lock all taskbars”, drag it to the top, optionally lock it back and profit!

Thanks

**Comment by radixs** [comment]
Okay that worked. I think in Win 10 I used for this taskbar settings, but in Win 11 even with ExplorerPatcher installed it sends me to Win 11 taskbar system settings page. That is why I made the request.
But of course, now it is not needed, thank you for the advice.

**Reply by valinet** [reply]
Great. Indeed, EP does not restore the old Settings app, so the option can only be activated like this or via manual registry edits. Also, I never enabled it via the Settings, so I didn’t even know there was a setting for that.
